ui/src/components/build/MobileClientBuildListItem.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import { DropdownKebab, MenuItem, Button, Row, Col} from 'patternfly-react';
+import BuildStatus from '../common/BuildStatus';
+import MobileListViewItem from '../common/MobileListViewItem';
+import BuildConfig from './BuildConfig';
+import ComponentSectionLabel from '../common/ComponentSectionLabel';
+
+
+const actions = () => (
+  <React.Fragment id="mobile-client-actions" pullRight>
+  <Button>
+    Start Build
+  </Button>
+  <DropdownKebab>
+    <MenuItem>Edit</MenuItem>
+    <MenuItem>Delete</MenuItem>
+  </DropdownKebab>
+</React.Fragment>
+);
+
+const buildConfig = {
```

Comment:
Only a question here, is there a benefit to defining a const here over a function defined in the component itself returning the jsx?

Reply:
This is just the way Jan mocked the data for now. Once we have the backend hooked up this will be gone completely and props will be passed to this component.

Reply:
Typical the one I clicked on to leave the comment, I meant to call out either `const actions` or `const heading`, is there any benefit to having them here?